12_upgrade/60_technique_upgrade.txt

```diff
@@ -4,36 +4,88 @@ At the first installation, Rudder will automatically deploy a Technique library
 `/var/rudder/configuration-repository/techniques` directory.
 
 When upgrading Rudder to another version, a new (updated) Technique library will be deployed
-in `/opt/rudder/share/techniques`, and Rudder will automatically take care of updating the system
+in `/opt/rudder/share/techniques`, and Rudder will only update the *system*
 Techniques in the configuration-repository directory.
 
-However, the other Techniques will not be updated automatically (yet), so you will have to do
-it yourself.
+However, the other Techniques are not updated automatically in Rudder 4.x, so it needs a user action.
+
+
+
+[TIP]
+
+====
+
+If you want to do any modification in the Technique library, it is recommanded to create new versions of the Technique, and do your changes in this new versions to simplify update.
```

Comment:
"If you want to do any modification in a Technique from the standard library, it is recommanded to create a whole new Technique (copy the source one with a new name and version), and do your changes in this new Technique. Thus, your standard library won't diverge from Rudder upstream one, and you will be able to benefit from its updates. "